google-cloud-spanner/src/main/java/com/google/cloud/spanner/Backup.java

```diff
@@ -183,6 +183,8 @@ static Backup fromProto(
         .setDatabase(DatabaseId.of(proto.getDatabase()))
         .setEncryptionInfo(EncryptionInfo.fromProtoOrNull(proto.getEncryptionInfo()))
         .setProto(proto)
+        .setMaxExpireTime(Timestamp.fromProto(proto.getMaxExpireTime()))
```

Comment:
Did we test with `null` max expire time and `null` referencingBackupsList?

Reply:
We won't get a null value over here. The addAll function checks for non null values-> https://github.com/protocolbuffers/protobuf/blob/master/java/core/src/main/java/com/google/protobuf/AbstractMessageLite.java#L408